x-pack/plugins/security_solution/cypress/e2e/dashboards/enable_risk_score.cy.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewRule } from '../../objects/rule';
+import {
+  ENABLE_HOST_RISK_SCORE_BUTTON,
+  ENABLE_USER_RISK_SCORE_BUTTON,
+  getRiskScoreLatestTransformId,
+  getRiskScorePivotTransformId,
+  RiskScoreEntity,
+  RISK_SCORE_DASHBOARDS_INSTALLATION_SUCCESS_TOAST,
+  RISK_SCORE_INSTALLATION_SUCCESS_TOAST,
+} from '../../screens/entity_analytics';
+import { createCustomRuleEnabled } from '../../tasks/api_calls/rules';
+import { cleanKibana } from '../../tasks/common';
+import { login, visit } from '../../tasks/login';
+import {
+  deleteRiskScore,
+  intercepInstallRiskScoreModule,
+  waitForInstallRiskScoreModule,
+} from '../../tasks/risk_scores';
+import { findSavedObjects } from '../../tasks/risk_scores/saved_objects';
+import { getTransformState } from '../../tasks/risk_scores/transforms';
+import { ENTITY_ANALYTICS_URL } from '../../urls/navigation';
+
+const spaceId = 'default';
+
+describe('Enable risk scores', () => {
+  before(() => {
+    cleanKibana();
+    login();
+    createCustomRuleEnabled(getNewRule(), 'rule1');
+    deleteRiskScore({ riskScoreEntity: RiskScoreEntity.host, spaceId, deleteAll: true });
+    deleteRiskScore({ riskScoreEntity: RiskScoreEntity.user, spaceId, deleteAll: true });
+
+    visit(ENTITY_ANALYTICS_URL);
+  });
+
+  after(() => {
+    deleteRiskScore({ riskScoreEntity: RiskScoreEntity.host, spaceId, deleteAll: true });
+    deleteRiskScore({ riskScoreEntity: RiskScoreEntity.user, spaceId, deleteAll: true });
+  });
+
+  it('shows enable host risk button', () => {
+    cy.get(ENABLE_HOST_RISK_SCORE_BUTTON).should('exist');
+    intercepInstallRiskScoreModule();
+    cy.get(ENABLE_HOST_RISK_SCORE_BUTTON).click();
+    waitForInstallRiskScoreModule();
+  });
+
+  it('starts installing host risk score', () => {
+    cy.get(ENABLE_HOST_RISK_SCORE_BUTTON).should('be.disabled');
+  });
+
+  it('should install host risk score successfully', () => {
+    cy.get(RISK_SCORE_INSTALLATION_SUCCESS_TOAST(RiskScoreEntity.host)).should('exist');
```

Comment:
In this test we are not doing any interaction with the UI everything is done using the API, hence, consider moving this to another tool or re-write the test.